codecov.yml

```diff
@@ -25,7 +25,7 @@ codecov:
 coverage:
   precision: 2
   round: down
-  range: 55..80
+  range: 60..85
```

Comment:
Just cosmetic stuff. Since we have coverage around 84-86% so better increase, otherwise most of the part would be close to the green range.